### PR TITLE
WKO 4.0: Update ItUsabilityOperatorHelmChart, ItOperatorRestart to V9 ClusterResource

### PR DESCRIPTION
**Pull request by xiancao**
WKO 4.0: Update ItUsabilityOperatorHelmChart, ItOperatorRestart to V9 ClusterResource

Jenkins result:
https://build.weblogick8s.org:8443/job/weblogic-kubernetes-operator-kind-new/12576/

### CODE REVIEW

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorRestart.java [hunk 1]
```diff
@@ -216,7 +217,9 @@ void testRestartOperatorAndVerifyScaling() {
    * restartVersion are updated, and the server pods are recreated, the server pods' weblogic.domainRestartVersion
    * label is updated, and the new credentials are valid and can be used to access WebLogic RESTful
    * Management Services.
+   * Disabled due to bug.
    */
+  @Disabled
```
Comment (anpanigr) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorRestart.java:222]: Why disabled in cluster-reference branch only. Can  tag the JIRA accordingly.  We should not disable the test due to existing BUG

Comment (xiancao) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/ItOperatorRestart.java]: re-enabled the test

**Review by anpanigr — APPROVED** [review]
lgtm

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ClusterUtils.java [hunk 2]
```diff
@@ -143,7 +143,7 @@ public static boolean scaleCluster(String clusterName, String namespace, int rep
   public static DomainResource addClusterToDomain(String clusterName, String namespace,
                                                   DomainResource domain, int replicas) {
     ClusterList clusters = listClusterCustomResources(namespace);
-    if (clusters.getItems().stream().anyMatch(cluster -> cluster.getClusterName().equals(clusterName))) {
+    if (clusters.getItems().stream().anyMatch(cluster -> cluster.getClusterResourceName().equals(clusterName))) {
```
Comment (sankarpn) [inline integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ClusterUtils.java:146]: Good catch :-)
Can you please change the method parameter from clusterName to clusterResourceName also to match with what is being checked?

**Review by sankarpn — APPROVED** [review]
lgtm

File: integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ClusterUtils.java (hunk 2 above)
Comment (xiancao) [reply integration-tests/src/test/java/oracle/weblogic/kubernetes/utils/ClusterUtils.java]: done

**Pull request merged**